tgstation.dme

```diff
@@ -1150,6 +1150,7 @@
 #include "code\modules\client\preferences.dm"
 #include "code\modules\client\preferences_savefile.dm"
 #include "code\modules\client\preferences_toggles.dm"
+#include "code\modules\client\verbs\etips.dm"

```

Comment:
I could have done that right now if you had kept them in the same file but since you didn't i can't use the web editor and i'm not making a whole branch for this.